lxd/instance/instance_interface.go

```diff
@@ -163,6 +163,7 @@ type Container interface {
 	ConsoleLog(opts liblxc.ConsoleLogOptions) (string, error)
 	InsertSeccompUnixDevice(prefix string, m deviceConfig.Device, pid int) error
 	DevptsFd() (*os.File, error)
+	IdmappedStorage(string) idmap.IdmapStorageType
```

Comment:
@brauner just a nit, but to help with documentation this should have the argument name (for example like InsertSeccompUnixDevice above).